global_costmap:
  global_frame: map
  robot_base_frame: base_footprint

  update_frequency: 1.0
  publish_frequency: 0.5
  transform_tolerance: 1

  static_map: true
 
